app/mailers/friend_notifier_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class FriendNotifierMailer < ApplicationMailer
+
+  def inform(user)
+  @user = user
+  #mail(to: friend_email, subject: "#{@user.first_name} is sending you an email")
+  mail(to: @user.email, subject: "#{@user.first_name} is sending you an email")
+  end
+end
```

Comment:
This is like a controller for an email, this is what you will see as the subject the email in your inbox

Reply:
So awesome! Email? This is like magic.